src/mixins/Projector.ts

```diff
@@ -215,14 +218,22 @@ export function ProjectorMixin<P, T extends Constructor<WidgetBase<P>>>(base: T)
 		}
 
 		public setChildren(children: DNode[]): void {
+			this._projectorChildren = [ ...children ];
```

Comment:
Why wouldn't we be doing `super.__setChildren__(this._projectorChildren)`?  Feels like a little bit of indirection.  Also if we did this, wouldn't we just be able to `this.setChildren(this.children)` and `this.setProperties(this.properties)` below?